<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
    <line>
      <![CDATA[Test start execution]]>
    </line>
    <line>
      <![CDATA[Configure the Report]]>
    </line>
    <line>
      <![CDATA[Database Connected]]>
    </line>
    <line>
      <![CDATA[Configure the Parallel Exe]]>
    </line>
    <line>
      <![CDATA[Navigated and logged in]]>
    </line>
    <line>
      <![CDATA[ON Test Start execution]]>
    </line>
    <line>
      <![CDATA[ OnTest Failure execution]]>
    </line>
    <line>
      <![CDATA[Logout]]>
    </line>
    <line>
      <![CDATA[Close the browser]]>
    </line>
    <line>
      <![CDATA[Configure the Parallel Exe]]>
    </line>
    <line>
      <![CDATA[Database Connection failed]]>
    </line>
  </reporter-output>
  <suite started-at="2025-06-17T17:44:02 IST" name="Default suite" finished-at="2025-06-17T17:44:13 IST" duration-ms="11180">
    <groups>
    </groups>
    <test started-at="2025-06-17T17:44:02 IST" name="Default test" finished-at="2025-06-17T17:44:13 IST" duration-ms="11180">
      <class name="ContactModule.ContactAllTestSenarios">
        <test-method is-config="true" signature="ConnectionWithDB()[pri:0, instance:ContactModule.ContactAllTestSenarios@7f485fda]" started-at="2025-06-17T17:44:01 IST" name="ConnectionWithDB" finished-at="2025-06-17T17:44:02 IST" duration-ms="317" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[Database Connected]]>
            </line>
          </reporter-output>
        </test-method> <!-- ConnectionWithDB -->
        <test-method is-config="true" signature="ConfigureParallelExe()[pri:0, instance:ContactModule.ContactAllTestSenarios@7f485fda]" started-at="2025-06-17T17:44:02 IST" name="ConfigureParallelExe" finished-at="2025-06-17T17:44:02 IST" duration-ms="1" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[Configure the Parallel Exe]]>
            </line>
          </reporter-output>
        </test-method> <!-- ConfigureParallelExe -->
        <test-method is-config="true" signature="LaunchTheBrowser()[pri:0, instance:ContactModule.ContactAllTestSenarios@7f485fda]" started-at="2025-06-17T17:44:02 IST" name="LaunchTheBrowser" finished-at="2025-06-17T17:44:03 IST" duration-ms="1209" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LaunchTheBrowser -->
        <test-method is-config="true" signature="Login()[pri:0, instance:ContactModule.ContactAllTestSenarios@7f485fda]" started-at="2025-06-17T17:44:03 IST" name="Login" finished-at="2025-06-17T17:44:05 IST" duration-ms="2398" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[Navigated and logged in]]>
            </line>
          </reporter-output>
        </test-method> <!-- Login -->
        <test-method signature="CreateConttName()[pri:0, instance:ContactModule.ContactAllTestSenarios@7f485fda]" started-at="2025-06-17T17:44:05 IST" name="CreateConttName" finished-at="2025-06-17T17:44:11 IST" duration-ms="6109" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.fail(Assert.java:116)
at ContactModule.ContactAllTestSenarios.CreateConttName(ContactAllTestSenarios.java:82)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[ON Test Start execution]]>
            </line>
            <line>
              <![CDATA[ OnTest Failure execution]]>
            </line>
          </reporter-output>
        </test-method> <!-- CreateConttName -->
        <test-method is-config="true" signature="Logout()[pri:0, instance:ContactModule.ContactAllTestSenarios@7f485fda]" started-at="2025-06-17T17:44:12 IST" name="Logout" finished-at="2025-06-17T17:44:13 IST" duration-ms="1113" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[Logout]]>
            </line>
          </reporter-output>
        </test-method> <!-- Logout -->
        <test-method is-config="true" signature="CloseTheBrowser()[pri:0, instance:ContactModule.ContactAllTestSenarios@7f485fda]" started-at="2025-06-17T17:44:13 IST" name="CloseTheBrowser" finished-at="2025-06-17T17:44:13 IST" duration-ms="156" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[Close the browser]]>
            </line>
          </reporter-output>
        </test-method> <!-- CloseTheBrowser -->
        <test-method is-config="true" signature="CloseParallelExe()[pri:0, instance:ContactModule.ContactAllTestSenarios@7f485fda]" started-at="2025-06-17T17:44:13 IST" name="CloseParallelExe" finished-at="2025-06-17T17:44:13 IST" duration-ms="0" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[Configure the Parallel Exe]]>
            </line>
          </reporter-output>
        </test-method> <!-- CloseParallelExe -->
        <test-method is-config="true" signature="CloseConnectionWithDB()[pri:0, instance:ContactModule.ContactAllTestSenarios@7f485fda]" started-at="2025-06-17T17:44:13 IST" name="CloseConnectionWithDB" finished-at="2025-06-17T17:44:13 IST" duration-ms="0" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[Database Connection failed]]>
            </line>
          </reporter-output>
        </test-method> <!-- CloseConnectionWithDB -->
      </class> <!-- ContactModule.ContactAllTestSenarios -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
